File: C#/array.cs
using System;
class program
{
    static void Main()
    {
        int n=Convert.ToInt32(Console.ReadLine());
        string[] input=Console.ReadLine().Split(' ');
        int[] arr=new int[n];
        for(int i=0; i<n ;i++)
        {
            arr[i]=Convert.ToInt32(input[i]);
        }
        int result=SimpleArraySum(arr);
        Console.WriteLine(result);
    }
    static int SimpleArraySum(int[] arr)
    {
        int total=0;
        for(int i=0; i<arr.Length; i++)
        {
            total+=arr[i];
        }
        return total;
    }
}
        


    

File: JAVA/array.java
import java.io.*;
import java.util.Scanner;

public class main
{
    public static void main(String[] args)
    {
        Scanner s=new Scanner(System.in);
        int n=s.nextInt();
        int[] arr=new int[n];
        for(int i=0;i<n;i++)
        {
            arr[i]=s.nextInt();
        }
        int sum=simpleArraySum(arr);
        System.out.println(sum);
    }
    public static int simpleArraySum(int[]arr)
    {
        int total=0;
        for (int i=0;i<arr.length;i++)
        {
            total+=arr[i];
        }
        return total;
    }
}
 
    